tests/test_b2b.py

```diff
@@ -19,3 +19,12 @@ def test_error() -> None:
     assert b2b is not None
     with pytest.raises(RuntimeError):
         b2b.flight_list(today, origin="LBO")
+
+
+@pytest.mark.skipif(b2b is None, reason="No key available")
+def test_forecasted_flights() -> None:
+    assert b2b is not None
+    res = b2b.forecasted_flights_list(
+        "2023-09-23 00:00", "2023-09-23 23:59", aerodrome="EGCC"
```

Comment:
This test will work today, but fail in a week...

Reply:
I've fixed it